endpoints.json

```diff
@@ -23,5 +22,74 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "returns an article object by its id",
+    "exampleResponse": {
+      "article": {
+        "article_id": 1,
+        "title": "Seafood substitutions are increasing",
+        "topic": "cooking",
+        "author": "weegembump",
+        "body": "Text from the article..",
+        "created_at": 1527695953341,
+        "votes": 100,
+        "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        "comment_count": 1
+      }
+    }
+  },
+  "GET /api/articles/:article_id/comments": {
+    "description": "returns an array of comments on an article by its id",
+    "exampleResponse": {
+      "comments": [
+        {
+          "comment_id": 1,
+          "votes": 100,
+          "created_at": 1527695953341,
+          "author": "name",
+          "body": "comment text",
+          "article_id": 1
+        }
+      ]
+    }
+  },
+  "POST /api/articles/:article_id/comments": {
+    "description": "adds a comment about the specified article",
+    "exampleResponse": {
+      "username": "butter_bridge",
+      "body": "an inspirational read"
+    }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "adjusts the number of votes on a comment",
+    "exampleResponse": {
+      "article": {
+        "article_id": 1,
+        "title": "Seafood substitutions are increasing",
+        "topic": "cooking",
+        "author": "weegembump",
+        "body": "Text from the article..",
+        "created_at": 1527695953341,
+        "votes": 100,
+        "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        "comment_count": 1
+      }
+    }
+  },
```

Comment:
This just as the above also needs an "exampleBody" which shows how/what the expect body should be i.e. an object with inv_votes